spec/models/text_widget_spec.rb

```diff
@@ -35,10 +35,12 @@
       expect(widget.text_body_html).to eq "123"
     end
 
-    it "should be invalid with a non-spec'd key" do
-      widget.content['not_a_real_field'] = "heyy"
-      expect(widget).not_to be_valid
-    end
+    ## Not sure this is necessary. Even if someone
+    # should access +content+ directly, the object
+    # won't have a getter, so the value would be hidden
+    #   widget.content["blah"] = "carrots"
+    #   widget.blah // raises undefined method
+    it "should be invalid with a non-spec'd key"
```

Comment:
I think it's important for the same reasons that Rails has always had attr_accessor or strong params - you don't want to let someone store arbitrary data on your models, you want to be able to enforce that you are only storing the expected data. Because strong params admits any arbitrary blob for content, it makes sense to push the enforcement up to the model that knows what keys it expects.